Post/views.py

```diff
@@ -27,6 +28,18 @@ def CreateNewPost(request):
     return render(request, 'Post/createPost.html', {'form': form})
 
 
+class PostDeleteView(LoginRequiredMixin, UserPassesTestMixin, DeleteView):
+    model = Post
+    template_name = 'Post/deletePost.html'
+    success_url = '/postList/'
+
+    def test_func(self):
```

Comment:
please rename this

Reply:
This function needed to be implemented so 'UserPassesTestMixin' will run it.